docs/UserGuide.md

```diff
@@ -146,6 +146,44 @@ Examples:
 * `changeqty 1 a/+1` increases the quantity of the 1st food item by 1.
 * `changeqty 2 a/-2` decreases the quantity of the 2nd food item by 2.
 
+### Sort food items by description: `sortdesc`
+
+Sorts the list of food items by description, then by expiry date from oldest to newest, then by priority from high to low. 
```

Comment:
Maybe here we can give a more brief idea? "Sorts the list of food items by description. In case of ties, it uses the expiry data and priority to break the ties.

Reply:
Something similar for sortpriority and sortexpiry can also be consider.

Reply:
The tie breaker sorting description is actually mentioned in subsequent lines. I think I will move your suggested line below.